lib/build.js

```diff
@@ -187,12 +188,23 @@ function getStatusConfig({ metadata }) {
             }
         }
 
+        const status = hoek.reach(checkObject, 'status', { default: 'PENDING' });
+
+        if (!['SUCCESS', 'FAILURE', 'PENDING'].includes(status)) {
```

Comment:
Can you use the data-schema: https://github.com/screwdriver-cd/data-schema/blob/master/plugins/scm.js#L354